frontend/src/Home.js

```diff
@@ -33,6 +33,10 @@ const NavLinkStyled = styled(NavLink)`
   justify-content: center;
   text-decoration: none;
   color: #12263d;
+
+  h4 {
```

Comment:
macht es Sinn, die hs ggf. alle in den GlobalStyles zu definieren?

Reply:
ja macht es